libbz2-rs-sys-cdylib/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "libbz2-rs-sys-cdylib"
-version = "0.0.0"
+readme = "./README.md"
 edition = "2021"
-readme = "README.md"
 license = "bzip2-1.0.6"
 repository = "https://github.com/trifectatechfoundation/libbzip2-rs"
 homepage = "https://github.com/trifectatechfoundation/libbzip2-rs"
```

Comment:
These can use workspace inheritance, right?

Reply:
the cdylib has its own local readme (with instructions on cargo-c etc)

Reply:
also cdylib is not in the workspace for feature flag reasons (the defaults are different, and workspaces build the union of all features)